packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+import { test, expect, Page, BrowserContext } from '@playwright/test';
+import configureI18n, { registerTranslationDir } from './configurei18n';
+
+if (process.env.LOCALE_DIR) {
+    registerTranslationDir(process.env.LOCALE_DIR);
+}
+
+// Configure i18n for english and french language. The playwright test is not
+// run in the browser context itself, so translations need to be manually loaded
+// if we want to identify widgets by their texts.
+
+// FIXME Let surveys define their own set of languages
+const i18n = configureI18n(['en', 'fr']);
+
+// Types for the tests
+type Value = string;
+type Text = string;
+type Url = string;
+type Title = string;
+type Path = string;
+type PathAndValue = { path: Path; value: Value };
+type HasTitleTest = ({ title }: { title: Title }) => void;
+type HasFrenchTest = () => void;
+type SwitchToEnglishTest = () => void;
+type HasConsentTest = () => void;
+type StartSurveyTest = () => void;
+type RegisterWithoutEmailTest = () => void;
+type HasUserTest = () => void;
+type InputRadioTest = ({ path, value }: PathAndValue) => void;
+type InputSelectTest = ({ path, value }: PathAndValue) => void;
+type InputStringTest = ({ path, value }: PathAndValue) => void;
+type InputRangeTest = ({ path, value }: {path: Path, value: number}) => void;
+type InputMapFindPlaceTest = ({ path }: { path: Path }) => void;
+type InputNextButtonTest = ({ text, nextPageUrl }: { text: Text; nextPageUrl: Url }) => void;
+
+let page: Page;
+let context: BrowserContext;
+let personId: string;
+const savedLabels: { [key: string]: number } = {}; // Get all the saved labels
+
+// Configure the tests to run in serial mode (one after the other)
+test.describe.configure({ mode: 'serial' });
+
+// Open the browser before all the tests and go to the home page
+test.beforeAll(async ({ browser }) => {
+    context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('/');
+
+    // Listen to console logs to get the personId
```

Comment:
Can you put a TODO. This part is a bit of a hack that I used.
TODO: Use a better method to get the personID

Reply:
Oui, c'était ma prochaine tâche, d'essayer d'ajouter un cas de test pour un ménage à 2 personnes...

Reply:
En effet, pour le moment ce n'est pas possible. J'ai hâte de voir ta solution.

Reply:
Voir https://github.com/chairemobilite/evolution/pull/493/commits/d0a7d4fb0c67963c67dc6f3a0a39997962064431 pour la solution avec test. Probablement pas parfait, mais sur une bonne voie.